src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let operationsCounter = 0;
+  const START_VALUE = 3;
+
+  const ERROR_MESSAGE = 'Bzzz... Error!';
+
+  return (num1) => {
+    operationsCounter++;
+
+    const isBigger = operationsCounter > START_VALUE;
```

Comment:
[What is bigger?](https://github.com/mate-academy/js_sum-with-closure/pull/2286#discussion_r1298270466)
It is necessary to choose a more meaningful name: `isOperationsLimitReached` or `isCallsLimitReached` or `isMaxCallsDone`.